ServerAdministration/DockerSwarm/DockerSwarm.md

```diff
@@ -0,0 +1,108 @@
+# Docker Swarm configuration
+
+The Docker Swarm is manged via the `docker-compose.yml` file that can be found in [GitHub](https://github.com/aau-giraf/giraf-production-swarm).
+
+The `docker-compose.yml` file consists of the following:
+```yml
+version: '3.7'
+
+services:
+    PROXY:                       # NGINX proxy service for the swarm
+        image: nginx:1.15        # This will use the latest version of 1.15.x
+        ports:                   # Exposes for 80 and 443 from the service to the swarm network(public)
+            - '80:80'            
+            - '443:443'          
+        volumes:                 # Mounts the nginx config folder inside the container as read only
+            - /swarm-nfs/nginx/:/etc/nginx/:r
+        networks:                # Attaches the network
+            - frontend
+        deploy:                  # Deploy options
+            replicas: 3          # Number er services
+        healthcheck:             # Docker healthcheck restarts of the nginx service is not running three times in a row, service might be out for a maximum of 30 secunds
+            test: ["CMD-SHELL", "service nginx status || exit 1"]
+            interval: 10s
+            timeout: 10s
+            retries: 3
+
+    API_PROD:                    # API service used for production
+        image: giraf/web-api:1   # This will use version 1 of the API hosted on hub.docker.com                                                             
+        networks:                # Attaches the network
+            - frontend
+            - backend
+        environment:             # Sets local envionment for dotnet
+            ASPNETCORE_ENVIRONMENT: Production
+        deploy:                  # Deploy options
+            replicas: 3          # Number of services
+        volumes:                 # Mounts the two NFS file shares into the container
+            - /swarm-nfs/cdn/pictograms/:/pictograms
+            - /swarm-nfs/api/appsettings.Develop.json:/srv/appsettings.json
+        healthcheck:             # Docker healthcheck restart of it fails three times in a row, service may be out for a maximum of 30 seconds
+            test: ["CMD-SHELL", "curl --fail http://localhost:5000 || exit 1"]
+            interval: 10s
+            timeout: 10s
+            retries: 3
+
+    API_DEV:                     # API service used for development
+        image: giraf/web-api:1   # This will use version 1 of the API hosted on hub.docker.com                                                             
+        networks:                # Attaches the network
+            - frontend
+            - backend
+        environment:             # Sets local envionment for dotnet
+            ASPNETCORE_ENVIRONMENT: Production
```

Comment:
Should `Production` be replaced with something else?

Reply:
No, production is simply a framework setting that forces the correct appsettings.json file to be loaded, I believe it would be more intuitive to keep an identical configuration for all API's